src/components/features/ctf-components/ctf-page-section/ctf-page-section.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { ComponentResolver } from '@src/components/shared/component-resolver';
+import { PageContainer } from '@src/components/templates/page-container';
+import { sectionTypes } from '@src/components/shared/global';
+
+interface PageSectionProps {
+  __typename: string;
+  sys: {
+    id: string;
+  }
+  componentsGrouped?: boolean;
+  componentsCollection?: any;
+  sectionType?: string;
+}
+
+
+export const CtfPageSection = (props: PageSectionProps) => {
+  const data = props.componentsCollection?.items as any[];
+
+  if (data && data.length > 0) {
```

Comment:
Je kunt ook gebruik maken van optional chaining: if (data?.length) en de > 0 kun je ook weglaten. Optional chaining gaat checken of data bestaat, zo niet gaat die undefined zijn, wat ervoor zorgt dat die in de if gecast wordt naar false. De > 0 mag ook weg, want 0 wordt ook gecast naar false. Da's kortere en leesbaardere code.